frontend/src/components/Settings/settings/tabs/DeveloperTab.tsx

```diff
@@ -440,6 +445,12 @@ export function DeveloperTab() {
             >
               <p className="text-lg font-main">invoice.paid</p>
             </Checkbox>
+            <Checkbox
```

Comment:
Todo: Add `style={{ marginLeft: "0px" }}` prop to `Checkbox` component as noted in #616